src/main/kotlin/moody/blackjack/domain/card/Score.kt

```diff
@@ -0,0 +1,46 @@
+package moody.blackjack.domain.card
+
+class Score private constructor(val value: Int) {
+
+    fun plus(score: Score): Score {
+        return from(this.value + score.value)
+    }
```

Comment:
```suggestion
     operator fun plus(score: Score): Score {
        return from(this.value + score.value)
    }
```
산술연산자를 오버로딩하는 방식으로 변경하면 + 연산을 사용하여 더할 수도 있습니다!

Reply:
와웅 또 하나 배워갑니다 워터 👍

Reply:
코틀린 인 액션에서 읽어놓고도 막상 쓸 수 있는 경우에 쓰질 못했군요ㅠㅠ 역시 책을 그냥 읽는 것에서 끝이 아니라 직접 손으로 써봐야 기억에 남는 거 같습니다 워터 덕분에 하나 배웠습니다ㅋㅋㅋㅋ